textos/Art.Opinion-IkerCastaño.md

```diff
@@ -0,0 +1,9 @@
+# Cuestión de educación
+
+## Es domingo. Son las dos del mediodía y ayer saliste de fiesta. Te acabas de despertar, te  estiras, piensas en la noche anterior (si recuerdas algo) y te levantas a comer con tu  familia.
```

Comment:
Entiendo que esto sería una entradilla: no es apropiado utilizar `##` (que sería para un encabezado de segundo nivel). Podéis destacar el texto usando `**` al principio y al final del texto que queréis destacar.